ProfessorAssessor/Views/ReviewCell.m

```diff
@@ -0,0 +1,33 @@
+@import Parse;
+#import "ReviewCell.h"
+#import "Course.h"
+
+static NSDateFormatter *dateFormatter = nil;
+
+@implementation ReviewCell
+
+- (void)setReview:(Review *)review {
+    [review.course
+     fetchIfNeededInBackgroundWithBlock:^
+     (PFObject *_Nullable object,
+      NSError *_Nullable error) {
+        if (object) {
+            Course *course = [Course courseFromPFObject:object];
+
+            self.courseName.text = course.name;
+            self.rating.value = [review.rating doubleValue];
+            self.content.text = review.content;
+
+            if (dateFormatter == nil) {
+                dateFormatter = [[NSDateFormatter alloc] init];
+                [dateFormatter setDateStyle:NSDateFormatterMediumStyle];
+                [dateFormatter setTimeStyle:NSDateFormatterNoStyle];
+            }
+
+            self.date.text = [dateFormatter
+                              stringFromDate:review.createdAt];
+        }
+    }];
+}
```

Comment:
If the rating is constantly being calculated, you would have to query the task each time to get the updated version, but maybe you could keep content and course as local variables in Review